project/profiles/capacity4more/modules/c4m/user/c4m_user_admin/c4m_user_admin.install

```diff
@@ -14,3 +14,10 @@ function c4m_user_admin_install() {
   $views_status['admin_views_user'] = TRUE;
   variable_set('views_defaults', $views_status);
 }
+
+/**
+ * Enables JQuery Migration plugin.
+ */
+function c4m_user_admin_update_7000() {
+  variable_set('jquery_update_jquery_migrate_enable', TRUE);
```

Comment:
there is no need to use updates yes since it is still not on production, move this to `capacity4more_setup_set_variables` (you can find it in `capacity4more.profile` file)